UI/js-src/lsmb/Reconciliation.js

```diff
@@ -0,0 +1,31 @@
+define("lsmb/Reconciliation",
+       ["dojo/_base/declare",
+        "dojo/topic",
+        "dojo/query",
+        "lsmb/Form",
+        "dijit/_Container",
+        "dojo/NodeList-dom",     // To load extensions in query
+        "dojo/domReady!"],
+       function(declare, Topic, Query, Form, _Container) {
+           return declare("lsmb/Reconciliation", [Form, _Container], {
+               update: function(targetValue, prefix) {
```

Comment:
lsmb/Reconciliation is a misnomer but it really inherit from lsmb/Form to specialize it and enable it to fold/unfold the cleared, mismatch and outstanding display sections.